www/site/content/docs/chef-workstation/getting-started.md

```diff
@@ -12,11 +12,11 @@ Chef Workstation gives you everything you need to get started with Chef. Ad-hoc
 
 ## Install Chef Workstation
 
-If you have not installed Chef Workstation, download and install it via https://www.chef.sh.
+If you have not installed Chef Workstation, download and install it from the [chef.sh](https://www.chef.sh) website.
 
 ## Check versions
 
-New ad-hoc commands `chef-run` and ChefDK commands such as `chef` are available via Chef Workstation. Your output may differ if you are running different versions.
+New ad-hoc commands `chef-run` and ChefDK CLI commands such as `chef` are available via Chef Workstation. See your installed version of Chef Workstation with `chef-run -v` and your installed version of the Chef tools with `chef -v`.
```

Comment:
Suggest: 

``You can also check your Workstation version by [platform-specific instructions for tray app] and choosing "About Chef Workstation". `